### PR TITLE
docs: Change latest version href

### LINKED ISSUE #2345: Create a `/latest` path in the docs
# What could be improved
I was following the tutorial on https://docs.flame-engine.org/1.6.0/tutorials/klondike/klondike.html (not this exact link, but my point stands exactly in this) and I saw a warning on top of the page that told me I needed to go to the latest docs because they might be outdated. There could be a path like `https://docs.flame-engine.org/latest/` so that people are not confused when starting with flame and finding it through Google or somewhere that could lead them to the older versions.

# Why should this be improved
So that when linking to the flame docs through a post on medium, a video description on youtube or anywhere the person will always go to the latest version.

### PR DESCRIPTION
**Pull request by immadisairaj**
The docs of the latest version say "1.7.0" will now go into the directory "latest" instead of "1.7.0". This commit will now point the docs to the new location.

This PR is a simultaneous work with flame-engine/flame-docs-site#15

## Checklist
- [x] I have followed the [Contributor Guide] when preparing my PR.
- [-] I have updated/added tests for ALL new/updated/fixed functionality.
- [-] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [-] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

This can be considered a breaking change. That is because, after this, the version `1.7.0` links won't work and will be replaced with `latest` links.

- [x] Yes, this PR is a breaking change.
- [-] No, this PR is not a breaking change.

## Related Issues
Closes #2345

### CODE REVIEW

**Comment by immadisairaj** [comment]
Currently, I haven't added the `robots.txt`. I will add that once I find time (maybe tomorrow).

File: doc/_sphinx/scripts/versions.js [hunk 1]
```diff
@@ -13,29 +13,46 @@ function getCurrentDocVersion() {
...
     let classes = 'btn btn-secondary topbarbtn';
     if (version === currentVersion) {
       classes += ' selected';
     }
-    out += `<a href="/${version}/">
-      <button class="${classes}">
-        <i class="fa fa-code-branch"></i> ${version}
-      </button>
-    </a>`;
+    // link the 'latest/` HTML path if the latest version
```
Comment (spydon) [inline doc/_sphinx/scripts/versions.js:36]:
```suggestion
    // Link to the 'latest/` path if it is the latest version.
```

**Review by spydon — APPROVED** [review]

**Comment by spydon** [comment]
> Currently, I haven't added the robots.txt. I will add that once I find time (maybe tomorrow).

I'm thinking that robots.txt should be added from flame-docs-site

**Comment by spydon** [comment]
> Currently, I haven't added the `robots.txt`. I will add that once I find time (maybe tomorrow).

Apparently `noindex` is the way to go instead of `robots.txt`:
https://developers.google.com/search/docs/crawling-indexing/block-indexing

**Comment by immadisairaj** [comment]
> Apparently `noindex` is the way to go instead of `robots.txt`:
> 
> https://developers.google.com/search/docs/crawling-indexing/block-indexing

Thanks for sharing the link. I will check it out.

**Pull request merged**